<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handleWidget</class>
 <widget class="QWidget" name="handleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="updateMemButton">
       <property name="text">
        <string>修改菜单权限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateAppButton">
       <property name="text">
        <string>修改app管理权限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateFabuButton">
       <property name="text">
        <string>修改发布权限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>修改cmdb管理权限</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="delRoleButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
